README.md

```diff
@@ -14,6 +14,7 @@ Uploadcare PHP integration handles uploads and further operations with files by
 * [Usage](#usage)
   * [Uploading files](#uploading-files)
   * [File operations](#file-operations)
+  * [Addons operations](#addons-operations)
```

Comment:
How about adding separate paragraph about Metadata? Maybe you can use some content from https://uploadcare.com/docs/file-metadata/ as a template.

Reply:
https://github.com/uploadcare/uploadcare-php/blob/update/api07/CHANGELOG.md#400